mrmustard/lab_dev/circuit_components_utils.py

```diff
@@ -97,3 +96,30 @@ def __init__(
     @property
     def representation(self) -> Bargmann:
         return Bargmann(*triples.displacement_map_s_parametrized_Abc(self.s, len(self.modes)))
+
+
+class _BtoQMap(CircuitComponent):
+    r"""The change of representation kernel from Bargmann into Quadrature as a ``Unitary``.
+
+    Used internally as a ``Unitary`` for transformations between representations on the ``Ket`` Wire.
+
+    The ``adjoint`` of this ``CircuitComponent`` denotes the change of representation kernel from Bargmann into Quadrature on the `Bra` Wire.
```

Comment:
```suggestion
    The ``adjoint`` of this ``CircuitComponent`` denotes the change of representation kernel from ``Bargmann`` into quadrature on the `bra` Wire.
```
Quadrature is not a class, so I'd leave it in small case everywhere (so, do a CTRL+F)